City-of-Helsinki/drupal-hdbt
# UHF-6839: Adress filter

# [UHF-6839](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6839)
Enables searching schools via address.

## What was done
Added features to the school search app:
* Functionality to search school by address
  * Queries coordinate data from api based on address
  * Queries unit IDs based on coordinate data
  * Sorts queries via geo distance query
* Added pagination to search results
* Added styles to the form component of the app
  * Moved some of event-list styles to the general react serach stylesheets for better general use

## How to install & test:

Check instructions here: https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/307

## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/307


[UHF-6839]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-6839?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
